src/Me/Modals/model.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-anonymous-default-export */
```

Comment:
Can't we [disable](https://stackoverflow.com/a/59249839/863110) this rule globally?

Reply:
Sure can.  I always forget that eslint rules can be managed in the `package.json`.  I was looking for an eslint config file, and of course not finding one.